nvme.c

```diff
@@ -407,20 +407,110 @@ static int get_ana_log(int argc, char **argv, struct command *cmd,
 	return nvme_status_to_errno(err, false);
 }
 
-static int get_telemetry_log(int argc, char **argv, struct command *cmd, struct plugin *plugin)
+static int get_telemetry_log_total_size_dalb4(int fd,
+					      struct nvme_telemetry_log *log,
+					      unsigned *total_size)
```

Comment:
Could you also change from `unsigned` to `unsigned int` when touching this code?

Reply:
Fixed as mentioned.